eos/models/wilson-coefficients.hh

```diff
@@ -168,23 +168,37 @@ namespace eos
          * These two operators are defined via
          *
          * H^eff = 4 G_F / sqrt(2) V_tb V_ts^* (
-         *             C_L alpha_e / (2 pi) * [sbar gamma^mu P_L b] [nubar gamma_mu P_L nu]
-         *             C_R alpha_e / (2 pi) * [sbar gamma^mu P_R b] [nubar gamma_mu P_L nu]
+         *             C_L alpha_e / (2 pi) * [sbar gamma^mu P_L b][nubar gamma_mu P_L nu] 
+         *             C_R alpha_e / (2 pi) * [sbar gamma^mu P_R b][nubar gamma_mu P_L nu] 
          *         )
-         *       = 4 G_F / sqrt(2) V_tb V_ts^* ( C_L O_L + C_R O_R )
+         *       = 4 G_F / sqrt(2) V_tb V_ts^* ( C_VLL O_VLL + C_VRL O_VRL )
+         * 
+         * Furthermore, we can add the scalar operators [FLS:2021A], eq. (2)
```

Comment:
I think this comment should include information that the new operators only emerge once you go beyond the SM content of particle, i.e., after including right-handed neutrino fields.

Reply:
I am still struggling a bit with fully understanding the physical meaning of all the individual operators. This is not extremely well described in the (paper)[https://arxiv.org/pdf/2111.04327.pdf]. I also had a look at (this)[https://link.springer.com/content/pdf/10.1007/JHEP09(2017)158.pdf]  and (this)[https://journals.aps.org/prd/pdf/10.1103/PhysRevD.60.094007] paper, but I am not yet very confident in my understanding. Do you have any good references that explain the physical consequences of the operators?
For example, I don't really see where the right-handed neutrino fields come in - because nu_L^c = (nu^c)_R corresponds to a RH antineutrino according to these (notes)[https://arxiv.org/pdf/hep-ph/0001264.pdf].

Reply:
You are mixing a couple of things here and I am not sure that Github is the best place for this discussion :) Let me try to give a quick answer and if you have further questions we can discuss them live.

- You are working in an effective theory, where all the heavy particles are "integrated out". The integrated fields can however still impact the physics at the low scale -- here the B decays -- indirectly. In the effective theory you therefore need to consider all the possible interactions (= operators) of your particles since some of these interactions may be enhanced by the heavy fields.

- In the SM, things are simple. You don't have right-handed neutrinos (so all the corresponding operators have a zero coefficient) and the effect of the SM heavy particles has been computed.

- You can now consider several NP scenarios: without or without right-handed neutrinos. Without RH neutrinos, you expect the NP heavy particles to modify the coefficients of your operators, by creating or modifying the interactions of your fields at the low scale. This is usually due to loop processes, where the NP particles can now appear. What your are studying is the sensitivity of the B->K(*) nu nu decay to these modifications.

- In other NP scenarios, right-handed neutrinos exist. If they have a small mass, they cannot be integrated out from your effective theory and you have to consider these fields even at the low-scale. These gives you new operators with non-zero coefficients. You are adding them as well because it is interesting to see if these new operators can be constrained by the future measurements.

- Finally, each operators involve its own fields with a specific chiralities and a given spin structure. They therefore have a different impact on the observable, especially on the angular distributions. So fitting the data in EOS allows to get a lot of information on the allowed ranges for the coefficients in the effective theory.